media/inf/intl.inf

```diff
@@ -770,9 +766,6 @@ LG_INDIAN              = "Indic"
 LG_GEORGIAN            = "Georgian"
 LG_ARMENIAN            = "Armenian"
 
-
-
-; Traditional Chinese
```

Comment:
Why removing the names of the languages corresponding to the sections?

Reply:
Because all other inf files also don't have it. Translators have to remember their language code anyway. Why having it here and not everywhere?